lib/orbit_web/controllers/rfid_controller.ex

```diff
@@ -0,0 +1,35 @@
+defmodule OrbitWeb.RfidController do
+  alias Orbit.BadgeSerial
+  use OrbitWeb, :controller
+  import Ecto.Query
+
+  alias Orbit.Repo
+
+  # TODO once we have permissions, require sign in permissions in order to use this endpoint
+
+  @spec show(Plug.Conn.t(), map()) :: Plug.Conn.t()
+  def show(conn, %{"badge_serial" => badge_serial}) do
+    badge_number =
+      Repo.one(
```

Comment:
This will crash if a badge serial isn't unique. Which is fine for now cuz we want to fix that anyway.